aerogear-bom/pom.xml

```diff
@@ -363,13 +363,7 @@
 
             <dependency>
               <groupId>org.keycloak</groupId>
-              <artifactId>keycloak-model-sessions-mem</artifactId>
-              <version>${keycloak.version}</version>
-            </dependency>
-
-            <dependency>
-              <groupId>org.keycloak</groupId>
-              <artifactId>keycloak-model-sessions-jpa</artifactId>
+              <artifactId>keycloak-model-sessions-infinispan</artifactId>
               <version>${keycloak.version}</version>
```

Comment:
yeah, I think having support for that is nice.

See: https://aerogear.org/docs/unifiedpush/ups_userguide/index/#cluster

On Thu, Nov 26, 2015 at 12:53 PM, Bruno Oliveira <notifications@github.com>
wrote:

> In aerogear-bom/pom.xml
> <https://github.com/aerogear/aerogear-parent/pull/53#discussion_r45970786>
> :
>
> > @@ -363,13 +363,7 @@
> >
> >              <dependency>
> >                <groupId>org.keycloak</groupId>
> > -              <artifactId>keycloak-model-sessions-mem</artifactId>
> > -              <version>${keycloak.version}</version>
> > -            </dependency>
> > -
> > -            <dependency>
> > -              <groupId>org.keycloak</groupId>
> > -              <artifactId>keycloak-model-sessions-jpa</artifactId>
> > +              <artifactId>keycloak-model-sessions-infinispan</artifactId>
> >                <version>${keycloak.version}</version>
>
> @matzew <https://github.com/matzew> are we going to use UPS in a
> clustered environment? If yes, that's required, otherwise, I don't think so.
>
> Take a look:
> http://keycloak.github.io/docs/userguide/keycloak-server/html/clustering.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-parent/pull/53/files#r45970786>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf